.github/workflows/ci.yml

```diff
@@ -36,8 +36,12 @@ jobs:
         with:
           node-version: 18
           cache: 'pnpm'
-      - name: pnpm fetch
```

Comment:
Did you mean to remove the name here but leave it everywhere else?

Reply:
Yes. For some reason github action does not like it.